src/ChartLabels.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace LoadTestToolbox
+{
+    public class ChartLabels
```

Comment:
can we get rid of the default values and constructors? just have it be a simple data class?